apps/examples/smartfs_powercut/smartfs_powercut_main.c

```diff
@@ -394,100 +498,153 @@ static int recovery_random_file(char *src, char* backup, char* backup_dir, int b
 	return ret;
 }
 
-static int recovery_file(char *src, char* backup, char* backup_dir, int bufsize, int idx, file_type_t type)
-{
-	if (type == FILE_TYPE_RANDOM) {
-		return recovery_random_file(src, backup, backup_dir, bufsize, idx);
-	}
-	/* TODO recovery proper text type..maybe it will memcpy from text. */
-	return OK;
-}
-
 int init_test_file(void)
 {
 	struct stat st;
 	int ret;
+	bool test_txt = false;
```

Comment:
Both sizes are being tested, this flag is for testing type of file
Random files are always tested
This acts as a flag to check if text files have to be tested.
because it depends on if user provides sample_file.txt

Reply:
Modified, obsolete now.
Inclusion  of Kconfig to chose file type and buffer size may affect this.